Makefile

```diff
@@ -54,16 +54,26 @@ define make-xc-target
   else
 		@printf "%s%20s %s\n" "-->" "${1}/${2}:" "${PROJECT}"
 		case "$2" in \
-			arm) export CGO_ENABLED="1" ; \
+			arm5) export CGO_ENABLED="1" ; \
```

Comment:
These all fail to compile for me with...

`arm-linux-gnueabi-gcc: error: unrecognized command line option '-m64'`

Disabling CGO (`export CGO_ENABLED="0"`) fixes it.

We could disable CGO as the bug that required that has been fixed (and theirs even a PR to remove it, #1471), but I'm curious if it did work for you and if so if you knew why? I've been digging around but haven't found anything useful yet.